src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/model/person/Person.kt

```diff
@@ -108,18 +121,24 @@ data class Person(
           pncIdentifier = defendant.pncId,
           croIdentifier = CROIdentifier.from(defendant.croNumber),
         ),
-        givenName = defendant.personDefendant?.personDetails?.firstName,
-        familyName = defendant.personDefendant?.personDetails?.lastName,
-        middleNames = defendant.personDefendant?.personDetails?.middleName?.split(" "),
-        dateOfBirth = defendant.personDefendant?.personDetails?.dateOfBirth,
+        names = Names(
+          preferred = Name(
+            title = defendant.personDefendant?.personDetails?.title,
+            firstName = defendant.personDefendant?.personDetails?.firstName,
+            lastName = defendant.personDefendant?.personDetails?.lastName,
+            middleNames = defendant.personDefendant?.personDetails?.middleName,
+            dateOfBirth = defendant.personDefendant?.personDetails?.dateOfBirth,
+            type = NameType.PREFERRED,
+          ),
+          aliases = defendant.aliases?.map { Name.from(it) } ?: emptyList(),
```

Comment:
If we ever want to make an `alias` the `preferred` name this structure might make it more difficult? As far as I know this is not a requirement though